session/manager_test.go

```diff
@@ -0,0 +1,17 @@
+package session
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestManagerHasUniqueSessionsStored(t *testing.T) {
+	manager := Manager{}
+	length := 10
+
+	for i := 0; i < length; i++ {
```

Comment:
Looks like loop with overhead. 
You can:
- create SID
- test private function `sessionIsUnique()`